core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -170,7 +171,16 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
     * Tests the ability of producing and consuming with the appropriate ACLs set.
     */
   @Test
-  def testProduceConsume {
+  def testProduceConsumeViaAssign {
```

Comment:
Hi @fpj I was actually working on 
[https://issues.apache.org/jira/browse/KAFKA-3396](url)

To test that the behavior when DESCRIBE authorization is missing is comparable to an unknown topic, I needed to test both paths the consumer can take.
And then I found that the `EndToEndAuthorizationTest` was failing on subscribe, regardless of the authorization setting, so i opened this bug.